### DISCUSSION TITLE
How to show a form in Modal?

### DISCUSSION THREAD

**Discussion by yogeshvachhani** [Budibase/budibase#10223] [Help] [answered]
Hi,

I am just getting started with BB.

But I got held up while trying to convert an existing web app.

In this app we use a lot of Modal Dialogs Forms that popup when user clicks on various buttons.

Here is a sample of what I want to achieve.

![image](https://user-images.githubusercontent.com/109499572/230113692-ee83a9c2-0d19-40f8-b06e-57cb2c6c7102.png)

I scanned the docs but could not find any reference to Modal Dialogs or PopUps.

Regards,

Yogesh

**Comment by melohagan** [comment, marked as answer]
Hey @yogeshvachhani 

This may help! https://docs.budibase.com/docs/screens#modal-screen

You just create a screen that you want to be your modal, then use the *Open screen in modal* checkbox to pop up.

The [Side panels](https://docs.budibase.com/docs/side-panel) are also an alternative option!
